package.json

```
@@ -65,7 +65,14 @@
     "postinstall": "node ./scripts/swap-package-links.mjs",
     "changelog": "bin/changelog",
     "publish": "cd tmp && npm publish && cd ..",
-    "publish-all": "npm run all publish"
+    "publish-all": "npm run all publish",
+    "examples:version": "node scripts/code-examples.js version",
+    "examples:install": "node scripts/code-examples.js install",
+    "examples:build": "node scripts/code-examples.js build",
+    "examples:build:ci": "cross-env NODE_ENV=gh-actions node scripts/code-examples.js build",
+    "examples:test": "node scripts/code-examples.js test",
+    "examples:test:ci": "cross-env NODE_ENV=gh-actions node scripts/code-examples.js test",
```

We can use https://docs.github.com/en/actions/reference/environment-variables#about-environment-variables and define ENV in GH Action workflow file

```suggestion
    "examples:build": "node scripts/code-examples.js build",
    "examples:test": "node scripts/code-examples.js test",
```

---

Done https://github.com/handsontable/handsontable/pull/7515/commits/3e69e96f3860737d57c4cf8706250ba9d9b67c0d